### PR TITLE
Update IN-SO emission factors

### PR DESCRIPTION
**Pull request by unitrium**
## Issue

Unknowns are partly aggregated renewables, for which the emission factor should be much lower.

## Description

Apply the same emission factor as IN-NO to IN-SO for unkowns.

### Double check

- [ ] I have tested my parser changes locally with `poetry run test_parser "zone_key"`
- [ ] I have run `pnpx prettier --write .` and `poetry run format` to format my changes.

### CODE REVIEW

**Comment by VIKTORVAV99** [comment]
I have already created a fix for the failing test at #5446

File: config/zones/IN-SO.yaml [hunk 1]
```diff
@@ -18,6 +18,21 @@ contributors:
...
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 45
+  lifecycle:
+    unknown:
+      source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
+      value: 192
```
[the remark this answers is not in the document]
Comment (unitrium) [reply config/zones/IN-SO.yaml:29]: That would be nice, do you have an example of how to do it ?

[the remark this answers is not in the document]
Comment (VIKTORVAV99) [reply config/zones/IN-SO.yaml:29]: Here is the config for SE-SE1 as an example:
```yaml
emissionFactors:
  direct:
    biomass:
      datetime: '2021-01-01'
      source: BEIS 2021
      value: 0.0
    coal:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021
      value: 1057.262766
    gas:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021
      value: 447.9740216
    oil:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021
      value: 880.903938
    unknown:
      - _comment: Uses the 2020 (2022 edition) eurostat production values for SE for
          the sources in unknown
        _url: https://ec.europa.eu/eurostat/documents/38154/4956218/Energy-Balances-January-2022-edition.zip/2dc71f1b-806a-b9ed-e77d-67b7c705197b?t=1643660477829
        datetime: '2022-01-01'
        source: assumes 98.26% biomass, 1.09% oil, 0.64% peat, 0.01% coal
        value: 16.4
  lifecycle:
    biomass:
      datetime: '2014-01-01'
      source: BEIS 2021; IPCC 2014
      value: 230.0
    coal:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021; Oberschelp, Christopher, et al. "Global emission
        hotspots of coal power generation."
      value: 1153.629975
    gas:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021; IPCC 2014
      value: 567.9740216
    hydro:
      datetime: '2020-01-01'
      source: UNECE 2022
      value: 10.7
    nuclear:
      datetime: '2020-01-01'
      source: UNECE 2022
      value: 5.13
    oil:
      datetime: '2021-01-01'
      source: EU-ETS, ENTSO-E 2021; IPCC 2014
      value: 1124.903938
    solar:
      datetime: '2014-01-01'
      source: IPCC 2014
      value: 45.0
    unknown:
      - _comment: Uses the 2020 (2022 edition) eurostat production values for SE for
          the sources in unknown
        _url: https://ec.europa.eu/eurostat/documents/38154/4956218/Energy-Balances-January-2022-edition.zip/2dc71f1b-806a-b9ed-e77d-67b7c705197b?t=1643660477829
        datetime: '2022-01-01'
        source: assumes 98.26% biomass, 1.09% oil, 0.64% peat, 0.01% coal
        value: 245.7
    wind:
      datetime: '2014-01-01'
      source: UNECE 2022, WindEurope "Wind energy in Europe, 2021 Statistics and the
        outlook for 2022-2026" Wind Europe Proceedings (2021)
      value: 12.62
 ```

[the remark this answers is not in the document]
Comment (unitrium) [reply config/zones/IN-SO.yaml:29]: Thanks!

File: config/zones/IN-SO.yaml [hunk 2]
```diff
@@ -18,6 +18,23 @@ contributors:
   - gopikrishna1793
   - unitrium
 timezone: Asia/Kolkata
+emissionFactors:
+  direct:
+    unknown:
+      - datetime: '2020-01-01'
+        source: CEA 2022; assumes 56% biomass, 36% hydro, 8% unknown
```
Comment (jayaddison) [inline config/zones/IN-SO.yaml:25]: @unitrium do you have a URL reference for a Central Electricity Authority page / document where the Y2022 report can be found?

If possible I'd like to add it while attempting to move as many of the `source` references as I can into top-level zone `sources` entries as part of #6001 (generally both a source name and a `link` attribute for it seem like best-practice).

**Pull request merged**